docs/DeveloperDocumentation.rst

```diff
@@ -0,0 +1,111 @@
+   
+   * Create a database citesphere using this command. ``create database citesphere;`` press enter.
+   
+   * Use newly created database. ``use citesphere;`` press enter. You will get a message saying ``Database changed``.
+
+
+* Start Apache Kafka and Zookeeper.
+   * To start Zookeeper open terminal and use this command. ``sudo /Users/your_path/zookeeper-3.3.6/bin/zkServer.sh start``. 
+   * To start Kafka use this ``sudo /Users/your_path/kafka_2.11-1.1.1/bin/kafka-server-start.sh /Users/your_path/kafka_2.11-1.1.1/config/server.properties``  (there is space in between start.sh and /Users)
+
+**Perform the following steps after starting Apache Kafka and Zookeeper:**
+
+1. Clone the citesphere repository in local from ``https://github.com/diging/citesphere``
+
+2. Import the project in eclipse → While importing make sure the import is done from the directory - ``{directory where the project is cloned}/citesphere``
+
+3. Open the ``Servers view`` (Window → Show View → Servers, or maybe ``Window → Show View → Other`` ...  and select ``Server``, then ``Open``) and click ``New →  Server``. 
+
+4. Now select ``Tomcat v8.5 Server`` and click ``Next``. Select the Tomcat installation directory and click ``Finish``.
+
+5. In the ``Servers`` view, it can be noted that the ``Tomcat v8.5 Server at localhost`` will be present. Double click on this. Click on the Timeouts section. Specify the start time limit as 500s and stop time limit as 15s and press ``Command + S/Ctrl + S`` to save.
+
+6. Right click on the citesphere project ``Maven → Select Maven Profiles``...Select the ``profile id`` and ensure that the id is same as the id specified in the ``settings.xml`` file in ./m2 folder. Click ``Ok``.
+
+7. In the servers view, right click on the ``Tomcat v8.5 Server at localhost`` and select ``Add and Remove``...If citesphere is not added, add the same and click Finish.
+
+8. Right click on ``Tomcat v8.5 Server at localhost`` and click ``Clean``....
+
+9. Right click on ``Tomcat v8.5 Server at localhost`` and click ``Start``....
+
+10. Once the server is started up and running, go to ``http://localhost:8080/citesphere`` and verify if the citesphere page is loaded.
+
+11. Now, click on ``Sign Up`` at the bottom and create a new user with your credentials.
+
+12. Then logout and login with the credentials username: admin and password: admin.
+
+13. Click on ``Users`` tab. The user that was created should appear here. Add the user and click on ``Make Admin``. Now, you can logout and login back with your own credentials.
+
+14. Click on ``Connect Zotero``. It will redirect the page to the Zotero application. Click on ``Accept defaults``.
+
+15. Now in the MySQL shell, you can find all the citesphere tables by using this command ``show tables;``
+
+16. Similarly in MongoDB Compass, click on connect (need not specify any credentials). The list of collections in citesphere can be seen.
```

Comment:
remove

Reply:
Add another point:
After you ensure the correct key settings click "Save Key". You will be redirected to Citesphere where your groups should now show up.

Reply:
removed points 15 and 16, and added the point suggested